en/04_Changelogs/5.0.0.md

```diff
@@ -100,6 +130,15 @@ This is a major release and as a result there are a number of breaking API chang
 - `SilverStripe\Core\Extensible::invokeWithExtensions()` and `SilverStripe\Core\Extensible::extend()` now use the splat operator instead of having a concrete number of possible arguments.
 - The constructor for `SilverStripe\Versioned\Versioned` now explicitly only accepts mode as a single argument.
 
+### Email {#api-email}
+
+- Email was sent in CMS 4 using [SwiftMailer](https://swiftmailer.symfony.com/docs/introduction.html). In CMS 5 this has been replaced with [symfony/mailer](https://symfony.com/doc/current/mailer.html).
+- `symfony/mailer` is the currently maintained email package from Symfony, and it also a more flexible email system that allows easier integration with third-party email providers
```

Comment:
```suggestion
- Email was sent in CMS 4 using [SwiftMailer](https://swiftmailer.symfony.com/docs/introduction.html), which has since [gone End Of Life](https://symfony.com/blog/the-end-of-swiftmailer). In CMS 5 this has been replaced with [symfony/mailer](https://symfony.com/doc/current/mailer.html). `symfony/mailer` is the currently maintained email package from Symfony. It's a more flexible email system that allows easier integration with third-party email providers.
```

Reply:
Updated